runtime/j9vm/jvm.c

```diff
@@ -4259,8 +4259,12 @@ JVM_Open(const char* filename, jint flags, jint mode)
 #endif
 #endif
 
-#if defined(J9UNIX) || defined(J9ZOS390)
+#if defined(J9UNIX) || defined(J9ZOS390) || defined(OSX)
+#if defined(OSX)
+#define EXTRA_OPEN_FLAGS 0
+#else
 #define EXTRA_OPEN_FLAGS O_LARGEFILE
+#endif
```

Comment:
Should this just be a new ifdef for OSX?  The nesting seems to serve no purpose.